packages/sync-actions/test/utils/common-actions.spec.js

```diff
@@ -116,6 +112,120 @@ describe('Common actions', () => {
         ])
       })
     })
+    describe('`shouldUnsetOmittedProperties`', () => {
+      test('if true => should generate unset actions for omitted properties', () => {
```

Comment:
Nit: I'd not mimic argument naming in a test but describe intended behaviour instead. I assume this test suite doesn't follow that pattern much but for this it would mean something like:

```js
describe('omitting properties', () => {
  describe('when omitted properties should be unset', () => {});
  describe('when omitted properties should not be unset', () => {});
});
```

The advantage I see would be that tests are less coupled to implementation and can act more as an independent specification of it.